### ISSUE TITLE
Can't run bash scripts

### ISSUE THREAD

**Issue by oyuskiv** [xerial/sbt-pack#145]
Can't run bash script if mapped program name contains space character.  bat script works correctly. Names of scripts don't contain space character. 

There is small example of build.sbt file that leads to run REPL instead of run program.

```
packMain := Map("hello scala" -> "Test")
lazy val test = (project in file("."))
  .enablePlugins(PackPlugin)
  .settings(
    name := "Test app",
    version := "0.1",
    scalaVersion := "2.12.7",
    )
```

**Comment by xerial** [comment]
Generally speaking, program names should not contain any white space. I’m not sure why you need to run a command with a white space.

**Comment by oyuskiv** [comment]
I agree with you about  white space and this issue has very low priority imho. I made report only to notify you about this.

**Comment by xerial** [comment]
ok. We can sanitize the command name if it contains white spaces for more safety. We will not directly support command names with white spaces, though.
